src/renderer/stores/Workspace.ts

```diff
@@ -59,6 +61,14 @@ export class Workspace {
       this.showSidebar = !this.showSidebar;
       return false;
     });
+    Mousetrap.bind(["command+o", "ctrl+o"], () => {
+      this.openProject();
+      return false;
+    });
+    //Listen for event from main process. 
+    ipcRenderer.on('openProject', () => {
+      this.openProject();
```

Comment:
Let's close the project here as well ?

Reply:
This would be solved if we put the logic within `openProject()`, as mentioned above.